test-one.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+version=$1
+fnm install $version
+fnm exec --using=$version node test.js
+fnm exec --using=$version node --es_staging test.js
+fnm exec --using=$version node --harmony test.js
+fnm uninstall $version  # Don't fill the disk with node installs.
```

Comment:
```suggestion
nvm install $version
node test.js
node --es_staging test.js
node --harmony test.js
nvm uninstall $version  # Don't fill the disk with node installs.
```
`nvm install` auto-uses the version.

Reply:
The uninstall trick doesn't seem to matter, acutally; the result is still cached. I'm running it now and if it passes, then I might just `@actions/cache` `nvm cache dir` or something, if it fits.